### PR TITLE
feat: add green theme as background to docs

### PR DESCRIPTION
**Pull request by pratik151192**
Added green background to docs. The RGB for the background gradient has been taken from @FractalJackal to make sure its consistent with the gradient across other websites. 

Screenshot 2024-03-25 at 7 31 21 PM
Screenshot 2024-03-25 at 7 31 29 PM

### CODE REVIEW

**Comment by rishtigupta** [comment]
If we make this change, we need to ensure all pages are readable. Example, the following page looks rather odd since it makes it difficult to read the sdk icons:

Screenshot 2024-03-25 at 3 37 54 PM

**Comment by pratik151192** [comment]
@rishtigupta thanks for checking the icons page! I did check code and tables etc looked okay didnt notice this one. Will take a look.

**Comment by rishtigupta** [comment]
I feel this change might have messed up the margins for "Ask Mo":
Screenshot 2024-03-25 at 3 44 54 PM

The production website looks to have the correct margins:
Screenshot 2024-03-25 at 3 45 46 PM

**Comment by cprice404** [comment]
> If we make this change, we need to ensure all pages are readable. Example, the following page looks rather odd since it makes it difficult to read the sdk icons:
> Screenshot 2024-03-25 at 3 37 54 PM

FWIW I think these icons are already kinda messed up in dark mode:

image

 we should fix them but idk if we need to block this PR on it.

**Review by cprice404 — DISMISSED** [review]
i'm good with this when @allenheltondev and @rishtigupta are. thanks for doing it

File: src/css/custom.scss [hunk 1]
```diff
@@ -105,6 +105,45 @@ h4 {
...
+}
+
+/* Adjusting the navbar to blend with the background */
+/* Adjusting the navbar to blend with the new background gradient */
+/* Adjusting the navbar to blend with the background */
+.navbar {
+  background-image: linear-gradient(50deg, #0E2515, #25392b);
+  backdrop-filter: blur(5px); /* If you want a blur effect */
+  z-index: 100;
+  /* Other styles */
```
Comment (rishtigupta) [inline src/css/custom.scss]: Can we remove these comments from the css file?

**Comment by rishtigupta** [comment]
This change would enforce dark mode for our users. I think a fast follow-up PR would be nice to fix the sdk icons. 
I am good with the pr if the margins for "Ask Mo" are corrected. Thanks!

**Comment by pratik151192** [comment]
@rishtigupta @cprice404 I have already gotten the white SVGs from Hannah so I'll update them. I am not seeing the errors with margin on my end so not sure why you are seeing Rishti but will check

File: src/css/custom.scss (hunk 1 above)
Comment (pratik151192) [reply src/css/custom.scss]: ok

**Review by rishtigupta — APPROVED** [review]

**Pull request merged**